pinot-core/src/main/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -208,6 +208,8 @@ public long getLatestIngestionTimestamp() {
 
     int avgNumMultiValues = config.getAvgNumMultiValues();
 
+    _aggregateMetrics = config.aggregateMetrics();
```

Comment:
Let's put this line after line 173 since there are some lines that read variables from config.

Reply:
done.